Colibri.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Colibri : Ave, Muerte
+    {
+        //se sobre escribe metodo comer
+        public override void comer()
+        {
+            Console.WriteLine("el colibri come");
+        }
+        //se crea comportmiento diferente
+        public void volar()
+        {
+            Console.WriteLine("el colibri vuela");
+        }
+
+        //se reciben los parametros de muerte y se calcula cuanto vivio
+        public string Morir(int nacimiento, int faleccimiento, string causa)
+        {
+            int calculo;
+            calculo = nacimiento - faleccimiento;
+            Console.WriteLine("La fecha de su muerte es: " + faleccimiento, "vivio "+ calculo, "años y la causa de la muerte es"+ causa);
+            string m=Console.ReadLine();
+            return m;
+        }  
+        //se retorna el mismo string que recibimos
+        public string Reencarna(string revivio)
+        {
+            return revivio;
+        }
+        //creamos variable para guardar informacion del constructor        
+        ModeloAnimal guardarinfo;
```

Comment:
Global variables must be at the top of the class

Reply:
has been corrected